### PR TITLE
Removed direct references to jackson2 and used play libraries.

### PR DESCRIPTION
**Pull request by tzimisce012**
This will allow to customize the objectmapper

### CODE REVIEW

File: modules/swagger-codegen/src/main/resources/Baikal/newApiController.mustache [hunk 1]
```diff
@@ -6,17 +6,19 @@ package {{package}};
...
 import java.util.Map;
 import java.util.ArrayList;
-import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.google.inject.Inject;
 import java.io.IOException;
 import java.util.concurrent.CompletionStage;
-import java.util.concurrent.CompletableFuture;
 import swagger.SwaggerUtils;
+{{#isContainer}}
```
Comment (palmerabollo) [inline modules/swagger-codegen/src/main/resources/Baikal/newApiController.mustache:19]: Why this condition? What does `isContainer` mean?

Comment (tzimisce012) [reply modules/swagger-codegen/src/main/resources/Baikal/newApiController.mustache:19]: It was giving unused imports. It is only necessary if you are using TypeReference class. You can see this on line 56. 

This is unrelated with the reason of the commit though.

**Comment by palmerabollo** [comment]
@tzimisce012 @edrevo I don't like the idea of forking swagger-codegen. It will be another codebase to maintain. Shouldn't we try to contribute changes like this back to the community if we think they are useful?

**Review by palmerabollo — APPROVED** [review]

**Comment by tzimisce012** [comment]
We have tried, but the contributions are moving really slow. Also, one of the maintainers of play framework for java makes changes under his own criteria and this created issues with our generated code.

**Comment by tzimisce012** [comment]
One of the last contributions that I did to swagger. After 11 days I am still waiting that they accept the PR. 

https://github.com/swagger-api/swagger-parser/pull/495

**Comment by edrevo** [comment]
@palmerabollo, 100% agree. I only created the fork becuase it was the only possible way to produce something usable before leaving on vacation. We should move all the Baikal-specific stuff to a new repo and just have this repo depend on the official version of swagger-codegen.

**Pull request merged**